# Spanish translations for savanna.
# Copyright (C) 2014 ORGANIZATION
# This file is distributed under the same license as the savanna project.
#
# Translators:
# jose.ramirez.rk <jose.ramirez.rk@gmail.com>, 2014
msgid ""
msgstr ""
"Project-Id-Version:  Savanna\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2014-01-20 06:07+0000\n"
"PO-Revision-Date: 2014-01-17 06:20+0000\n"
"Last-Translator: openstackjenkins <jenkins@openstack.org>\n"
"Language-Team: Spanish "
"(http://www.transifex.com/projects/p/openstack/language/es/)\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 1.3\n"

#: savanna/openstack/common/eventlet_backdoor.py:140
#, python-format
msgid "Eventlet backdoor listening on %(port)s for process %(pid)d"
msgstr "Eventlet oculto escuchando en %(port)s para el proceso %(pid)d"

#: savanna/openstack/common/exception.py:103
msgid "Uncaught exception"
msgstr "Excepción no controlada"

#: savanna/openstack/common/excutils.py:62
#, python-format
msgid "Original exception being dropped: %s"
msgstr "Se está descartando excepción original: %s"

#: savanna/openstack/common/excutils.py:91
#, python-format
msgid "Unexpected exception occurred %d time(s)... retrying."
msgstr "La excepción inesperada ha ocurrido %d vez(veces)... reintentando."

#: savanna/openstack/common/fileutils.py:63
#, python-format
msgid "Reloading cached file %s"
msgstr "Volviendo a cargar el archivo en memoria caché %s "

#: savanna/openstack/common/gettextutils.py:252
msgid "Message objects do not support addition."
msgstr ""

#: savanna/openstack/common/gettextutils.py:261
msgid ""
"Message objects do not support str() because they may contain non-ascii "
"characters. Please use unicode() or translate() instead."
msgstr ""

#: savanna/openstack/common/lockutils.py:103
#, python-format
msgid "Could not release the acquired lock `%s`"
msgstr "No se ha podido liberar el bloqueo adquirido `%s`"

#: savanna/openstack/common/lockutils.py:168
#, python-format
msgid "Got semaphore \"%(lock)s\""
msgstr "Semáforo obtenido \"%(lock)s\""

#: savanna/openstack/common/lockutils.py:177
#, python-format
msgid "Attempting to grab file lock \"%(lock)s\""
msgstr "Intentando bloquear archivo \"%(lock)s\""

#: savanna/openstack/common/lockutils.py:187
#, python-format
msgid "Created lock path: %s"
msgstr "Candado creado ruta: %s"

#: savanna/openstack/common/lockutils.py:205
#, python-format
msgid "Got file lock \"%(lock)s\" at %(path)s"
msgstr "Se ha bloqueado el archivo \"%(lock)s\" en %(path)s"

#: savanna/openstack/common/lockutils.py:209
#, python-format
msgid "Released file lock \"%(lock)s\" at %(path)s"
msgstr "Se ha liberado el bloqueo de archivo \"%(lock)s\" en %(path)s"

#: savanna/openstack/common/lockutils.py:247
#, python-format
msgid "Got semaphore / lock \"%(function)s\""
msgstr "Semáforo / bloqueo obtenido \"%(function)s\""

#: savanna/openstack/common/lockutils.py:251
#, python-format
msgid "Semaphore / lock released \"%(function)s\""
msgstr "Semáforo / bloqueo liberado \"%(function)s\""

#: savanna/openstack/common/log.py:301
#, python-format
msgid "Deprecated: %s"
msgstr "En desuso: %s"

#: savanna/openstack/common/log.py:402
#, python-format
msgid "Error loading logging config %(log_config)s: %(err_msg)s"
msgstr "Error al cargar la configuración de registro %(log_config)s: %(err_msg)s"

#: savanna/openstack/common/log.py:453
#, python-format
msgid "syslog facility must be one of: %s"
msgstr "El recurso syslog debe ser uno de: %s"

#: savanna/openstack/common/log.py:623
#, python-format
msgid "Fatal call to deprecated config: %(msg)s"
msgstr "Llamada muy grave a configuración en desuso: %(msg)s"

#: savanna/openstack/common/loopingcall.py:82
#, python-format
msgid "task run outlasted interval by %s sec"
msgstr "la ejecución de tarea ha durado %s seg. más que el intervalo"

#: savanna/openstack/common/loopingcall.py:89
msgid "in fixed duration looping call"
msgstr "en llamada en bucle de duración fija"

#: savanna/openstack/common/loopingcall.py:129
#, python-format
msgid "Dynamic looping call sleeping for %.02f seconds"
msgstr "Llamada dinámica repetitiva, en espera por %.02f segundos"

#: savanna/openstack/common/loopingcall.py:136
msgid "in dynamic looping call"
msgstr "en llamada en bucle dinámica"

#: savanna/openstack/common/periodic_task.py:41
#, python-format
msgid "Unexpected argument for periodic task creation: %(arg)s."
msgstr "Argumento inesperado para la creación de tarea periódica: %(arg)s."

#: savanna/openstack/common/periodic_task.py:132
#, python-format
msgid "Skipping periodic task %(task)s because its interval is negative"
msgstr "Omitiendo la tarea periódica %(task)s porque el intervalo es negativo"

#: savanna/openstack/common/periodic_task.py:137
#, python-format
msgid "Skipping periodic task %(task)s because it is disabled"
msgstr "Omitiendo la tarea periódica %(task)s porque está inhabilitada"

#: savanna/openstack/common/periodic_task.py:175
#, python-format
msgid "Running periodic task %(full_task_name)s"
msgstr "Ejecutando tarea periódica %(full_task_name)s"

#: savanna/openstack/common/periodic_task.py:184
#, python-format
msgid "Error during %(full_task_name)s: %(e)s"
msgstr "Error durante %(full_task_name)s: %(e)s"

#: savanna/openstack/common/processutils.py:130
#, python-format
msgid "Got unknown keyword args to utils.execute: %r"
msgstr ""
"Se han obtenido argumentos de palabra clave desconocidos en "
"utils.execute: %r"

#: savanna/openstack/common/processutils.py:145
#, python-format
msgid "Running cmd (subprocess): %s"
msgstr "Ejecutando cmd (subprocesos): %s"

#: savanna/openstack/common/processutils.py:169
#: savanna/openstack/common/processutils.py:241
#, python-format
msgid "Result was %s"
msgstr "El resultado fue %s"

#: savanna/openstack/common/processutils.py:181
#, python-format
msgid "%r failed. Retrying."
msgstr "%r ha fallado. Volviendo a intentarlo."

#: savanna/openstack/common/processutils.py:220
#, python-format
msgid "Running cmd (SSH): %s"
msgstr "corriendo cmd (SSH): %s"

#: savanna/openstack/common/processutils.py:222
msgid "Environment not supported over SSH"
msgstr "Entorno no soportado a través de SSH"

#: savanna/openstack/common/processutils.py:226
msgid "process_input not supported over SSH"
msgstr "entrada de proceso no soporta a través de SSH"

#: savanna/openstack/common/service.py:166
#: savanna/openstack/common/service.py:384
msgid "Full set of CONF:"
msgstr "Conjunto completo de CONF:"

#: savanna/openstack/common/service.py:175
#: savanna/openstack/common/service.py:269
#, python-format
msgid "Caught %s, exiting"
msgstr "Se ha captado %s, saliendo"

#: savanna/openstack/common/service.py:187
msgid "Exception during rpc cleanup."
msgstr "Excepción durante limpieza de rpc."

#: savanna/openstack/common/service.py:238
msgid "Parent process has died unexpectedly, exiting"
msgstr "El proceso padre se ha detenido inesperadamente, saliendo"

#: savanna/openstack/common/service.py:275
msgid "Unhandled exception"
msgstr "Excepción no controlada"

#: savanna/openstack/common/service.py:308
msgid "Forking too fast, sleeping"
msgstr "Bifurcación demasiado rápida, en reposo"

#: savanna/openstack/common/service.py:327
#, python-format
msgid "Started child %d"
msgstr "Se ha iniciado el hijo %d"

#: savanna/openstack/common/service.py:337
#, python-format
msgid "Starting %d workers"
msgstr "Iniciando %d trabajadores"

#: savanna/openstack/common/service.py:354
#, python-format
msgid "Child %(pid)d killed by signal %(sig)d"
msgstr "Hijo %(pid)d matado por señal %(sig)d"

#: savanna/openstack/common/service.py:358
#, python-format
msgid "Child %(pid)s exited with status %(code)d"
msgstr "El hijo %(pid)s ha salido con el estado %(code)d"

#: savanna/openstack/common/service.py:362
#, python-format
msgid "pid %d not in child list"
msgstr "El pid %d no está en la lista de hijos"

#: savanna/openstack/common/service.py:392
#, python-format
msgid "Caught %s, stopping children"
msgstr "Se ha captado %s, deteniendo hijos"

#: savanna/openstack/common/service.py:410
#, python-format
msgid "Waiting on %d children to exit"
msgstr "En espera de %d hijos para salir"

#: savanna/openstack/common/sslutils.py:52
#, python-format
msgid "Unable to find cert_file : %s"
msgstr "No se puede encontrar cert_file: %s"

#: savanna/openstack/common/sslutils.py:55
#, python-format
msgid "Unable to find ca_file : %s"
msgstr "No se puede encontrar ca_file: %s"

#: savanna/openstack/common/sslutils.py:58
#, python-format
msgid "Unable to find key_file : %s"
msgstr "No se puede encontrar key_file: %s"

#: savanna/openstack/common/sslutils.py:61
msgid ""
"When running server in SSL mode, you must specify both a cert_file and "
"key_file option value in your configuration file"
msgstr ""
"Al ejecutar el servidor en modalidad SSL, debe especificar un valor para "
"las opciones cert_file y key_file en el archivo de configuración"

#: savanna/openstack/common/sslutils.py:100
#, python-format
msgid "Invalid SSL version : %s"
msgstr "Versión SSL inválida : %s"

#: savanna/openstack/common/wsgi.py:187
msgid "cannot understand JSON"
msgstr "no se puede entender JSON"

#: savanna/openstack/common/wsgi.py:211
msgid "cannot understand XML"
msgstr "no se puede entender XML"

#: savanna/openstack/common/crypto/utils.py:29
msgid "An unknown error occurred in crypto utils."
msgstr "Ha ocurrido un error desconocido en crypto utils."

#: savanna/openstack/common/crypto/utils.py:36
#, python-format
msgid "Block size of %(given)d is too big, max = %(maximum)d"
msgstr "Tamaño de bloque de %(given)d es demasiado grande, máximo = %(maximum)d"

#: savanna/openstack/common/crypto/utils.py:45
#, python-format
msgid "Length of %(given)d is too long, max = %(maximum)d"
msgstr "Longitud de %(given)d es demasiado larga, máxima = %(maximum)d"

#: savanna/openstack/common/db/exception.py:42
msgid "Invalid Parameter: Unicode is not supported by the current database."
msgstr "Parámetro no válido: la base de datos actual no soporta Unicode."

#: savanna/openstack/common/db/sqlalchemy/migration.py:191
msgid "version should be an integer"
msgstr "la versión debe ser un entero"

#: savanna/openstack/common/db/sqlalchemy/migration.py:221
msgid ""
"The database is not under version control, but has tables. Please stamp "
"the current version of the schema manually."
msgstr ""

#: savanna/openstack/common/db/sqlalchemy/session.py:596
msgid "DB exception wrapped."
msgstr "excepción DB empaquetada."

#: savanna/openstack/common/db/sqlalchemy/session.py:665
#, python-format
msgid "Database server has gone away: %s"
msgstr ""

#: savanna/openstack/common/db/sqlalchemy/session.py:743
msgid ""
"This application has not enabled MySQL traditional mode, which means "
"silent data corruption may occur. Please encourage the application "
"developers to enable this mode."
msgstr ""

#: savanna/openstack/common/db/sqlalchemy/session.py:767
#, python-format
msgid "SQL connection failed. %s attempts left."
msgstr "La conexión SQL ha fallado. Quedan %s intentos."

#: savanna/openstack/common/db/sqlalchemy/test_migrations.py:103
#, python-format
msgid "Got lock \"%s\""
msgstr ""

#: savanna/openstack/common/db/sqlalchemy/test_migrations.py:106
#, python-format
msgid "Lock released \"%s\""
msgstr ""

#: savanna/openstack/common/db/sqlalchemy/utils.py:58
msgid "Sort key supplied was not valid."
msgstr "La clave de clasificación proporcionada no es válida. "

#: savanna/openstack/common/db/sqlalchemy/utils.py:97
msgid "Id not in sort_keys; is sort_keys unique?"
msgstr "Id no está en sort_keys; ¿es sort_keys exclusivo?"

#: savanna/openstack/common/db/sqlalchemy/utils.py:119
msgid "Unknown sort direction, must be 'desc' or 'asc'"
msgstr "Dirección de clasificación desconocida, debe ser 'desc' o ' asc'"

#: savanna/openstack/common/db/sqlalchemy/utils.py:194
#, python-format
msgid ""
"Please specify column %s in col_name_col_instance param. It is required "
"because column has unsupported type by sqlite)."
msgstr ""
"Especifique la columna %s en el parámetro col_name_col_instance. Es "
"necesario porque la columna tiene un tipo no soportado por sqlite)."

#: savanna/openstack/common/db/sqlalchemy/utils.py:200
#, python-format
msgid ""
"col_name_col_instance param has wrong type of column instance for column "
"%s It should be instance of sqlalchemy.Column."
msgstr ""
"col_name_col_instance param tiene el tipo incorrecto de instancia de "
"columna para la columna %s. Debe ser una instancia de  sqlalchemy.Column."

#: savanna/openstack/common/db/sqlalchemy/utils.py:280
#, python-format
msgid "Deleting duplicated row with id: %(id)s from table: %(table)s"
msgstr "Eliminando registro duplicado con id: %(id)s de la tabla: %(table)s"

#: savanna/openstack/common/db/sqlalchemy/utils.py:301
msgid "Unsupported id columns type"
msgstr "Tipo de identificador de columnas no soportado"

